docs/dependent-apps.md

```diff
@@ -0,0 +1,289 @@
+         "targetCommit": "d43bea5e11658e653088ce4b9a91b6606c3c2a0d",
+         "env": null,
+         "targetEnv": null
+     }
+ ]
+```
+
+### POST /v1/devices
+Dependent Device Provision
+
+**Example**
+
+```bash
+curl -H "Content-Type: application/json" -X POST --data '{"appId": <appId>}' /
+$RESIN_SUPERVISOR_ADDRESS/v1/devices?apikey=$RESIN_SUPERVISOR_API_KEY
+```
+
+**Response**
+`HTTP/1.1 201 CREATED`
+
+
+```javascript
+{
+          "id": 47318,
+            "uuid": "8dc608765fd32665d49a268a7eb4657bc2ab8a56db06d2c57ef7c7e9a115da",
+      "name": "wild-paper",
+          "note": null,
+          "device_type": "edge",
+    }
+```
+
+### GET /v1/devices/:uuid
+Dependent Device Information
+
+**Example**
+
+```bash
+curl -X GET $RESIN_SUPERVISOR_ADDRESS/v1/devices/<uuid>?apikey=$RESIN_SUPERVISOR_API_KEY
+```
+
+**Response**
+`HTTP/1.1 200 OK`
+
+```javascript
+{
+           "id": 1,
+           "uuid": "5ae8cf6e062c033ea57837498ad9b487bfc714e9eab0fed0404ee56e397790",
+           "appId": 13015,
+           "device_type": "edge",
+           "logs_channel": "69f961abffaad2ff00031b29f718be4fb19e1bfabf1fee7a9ebfb5fa75a1fbdb",
+           "deviceId": "47270",
+           "is_online": null,
+           "name": "blue-sun",
+           "status": "Provisioned",
+           "download_progress": null,
+           "commit": "d43bea5e16658e623088je4b9a91b6606c3c2a0d",
+           "targetCommit": "d43bea5e16658e651088ce4b9a21b6606c3c2a0d",
+           "env": null,
+           "targetEnv": null
+       }
+```
+
+### PUT /v1/devices/:uuid
+Dependent Device Information Update
+
+**Example**
+
+```bash
+curl -H "Content-Type: application/json" -X PUT --data /
+'{"is_online":true, "status": "Updating", "commit": "339125a7529cb2c2a8c93a0bbd8af69f2d96286ab4f4552cb5cfe99b0d3ee9"}' /
+$RESIN_SUPERVISOR_ADDRESS/v1/devices/<uuid>?apikey=$RESIN_SUPERVISOR_API_KEY
+```
+
+**Response**
+`HTTP/1.1 200 OK`
+
+```javascript
+{
+           "id": 1,
+           "uuid": "5ae8cf6e062c033ea38437498ad9b482bcc714e9eab0fed0404ee56e397790",
+           "appId": 13015,
+           "device_type": "edge",
+           "logs_channel": "69f961abffaad2ff66031b29f712be4fb19e1bfabf1fee7a9ebfb5fa05a1fbdb",
+           "deviceId": "47270",
+           "is_online": true,
+           "name": "blue-sun",
+           "status": "Updating",
+           "download_progress": null,
+           "commit": "d43bea5e16658e653088ce4b9a11b6606c3c2a0d",
+           "targetCommit": "d43bea5e16658e653088se4b9a91b6606c3c2a0d",
+           "env": null,
+           "targetEnv": null
+       }
+```
+
+### POST /v1/devices/:uuid/logs
+Dependent Device Log
+
+**Example**
+
+```bash
+curl -H "Content-Type: application/json" -X POST --data '{"message":"detected movement","timestamp":1472142960}' /
+$RESIN_SUPERVISOR_ADDRESS/v1/devices/<uuid>/logs?apikey=$RESIN_SUPERVISOR_API_KEY
+```
+**Response**
+`HTTP/1.1 202 ACCEPTED`
+
+## Hook (the request the Resin Supervisor performs)
```

Comment:
All of this should probably go together with the Hook configuration above, instead of having the API endpoints explained in the middle.